examples/rosa-classic-public-with-shared-vpc/main.tf

```diff
@@ -0,0 +1,119 @@
+provider "aws" {
```

Comment:
Please add a README that describes all the steps for shared VPC

Reply:
Will do it in other PR, as par of the work on [OCM-4992](https://issues.redhat.com/browse/OCM-4992)